src/info.js

```diff
@@ -0,0 +1,39 @@
+const nqmUtils = require("@nqminds/nqm-core-utils");
+const jwt = require("jsonwebtoken");
+
+function getAccountInfo(api) {
+  const decoded = jwt.decode(api.accessToken);
+  return api.getAccount(decoded.sub);
+}
+
+function getServerFolderId(id) {
+  return nqmUtils.shortHash(
+    nqmUtils.constants.applicationServerDataFolderPrefix + id
+  );
+}
+
+async function getDatabotsIds(api) {
+  const filter = {
+    baseType: nqmUtils.constants.databotBaseType,
+  };
+  const result = await api.getResources(filter, {name: 1, id: 1, owner: 1}, {});
+  return JSON.stringify(result, 0, 2);
+}
+
+async function getInfo({api, id, type}) {
+  type = type || "";
+  id = id || "";
```

Comment:
We can use default options here:

```suggestion
async function getInfo({api, id = "", type = ""}) {
```

However, `c = c || default` would work for any false-y value, while the above is more like: `c = c !== undefined ? c : default`

Reply:
Done